styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -643,7 +646,7 @@ private static ServiceAccountKeyManager createServiceAccountKeyManager() {
     }
   }
 
-  static NamespacedKubernetesClient getKubernetesClient(Config rootConfig, String id,
+  public static NamespacedKubernetesClient getKubernetesClient(Config rootConfig, String id,
```

Comment:
This doesn't need to be `public`, it seems.

Reply:
👍